### DISCUSSION TITLE
What do rate limit errors from the API look like?

### DISCUSSION THREAD

**Discussion by dhughes** [weather-gov/api#772] [Q&A]
I'm trying to write a test for my application that mimics being rate limited by the weather.gov API. The documentation is a bit vague on what to expect. So, what's actually in a rate limit error? Does it have a 429 status? I've seen things that suggest that it might be a 403 error. Are there any headers telling us how long to back off? Is there anything predictable in the body of the response?

Any info will help. Thanks!

**Comment by sullynole** [comment]
Expect 403 response status error with a Reference ID # in the body.
